db/blip_handler.go

```diff
@@ -510,14 +520,10 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 					}
 
 				}
-				// if V3 and below populate change row with rev id
-				if bh.activeCBMobileSubprotocol <= CBMobileReplicationV3 {
-					for _, item := range change.Changes {
-						changeRow := bh.buildChangesRow(change, item["rev"])
-						pendingChanges = append(pendingChanges, changeRow)
-					}
-				} else {
-					changeRow := bh.buildChangesRow(change, change.CurrentVersion.String())
+				// if V3 and below populate change row with rev id, else populate with CV. If CV not available
+				// (pre upgraded doc) then rev will be sent
```

Comment:
This comment isn't really relevant to this specific area of code any more so can probably just be removed?

```suggestion
```